tasks/docs.rb

```diff
@@ -266,7 +266,6 @@ def build_resource_data(name, data)
       r = {}
 
       # These properties are always set to these values.
-      r["draft"] = false
       r["resource_reference"] = true
       r["robots"] = nil
       r["syntax_code_block"] = nil
```

Comment:
Does it still make sense to set these two values to nil?

Reply:
I think there's a bunch of these we can probably remove. I'm going to work with the docs team to figure out what we need to set and what we don't. Expect more PRs